acceptancetests/assess

```diff
@@ -162,14 +166,114 @@ def parse_args():
                               default=SERIES)
     pass_through.add_argument("--keep-env", action="store_true",
                               help="Preserve the testing directories, e.g. HOME_DIR, DATA_DIR, ... after the test completes")
+
     pass_through.add_argument("--logging-config",
-                              help="Override logging configuration for a deployment. [default: \"<root>=INFO;unit=INFO\"]",
+                              help="Override logging configuration for a deployment. [default: \"<root>=INFO;unit=INFO\" unless test defines its own defaults]",
                               default="<root>=INFO;unit=INFO")
+    pass_through.add_argument("--logging-module", help="Logger to use for the test. (Ignored by most tests)")
+    pass_through.add_argument("--charm-bundle", "--bundle", metavar="BUNDLE", dest="charm_bundle",
+                              help="Overwrite default bundle (Relevant only for deploy_lxd_profile_bundle, deploy_webscale tests)")
+    pass_through.add_argument("--charm-series", metavar="SERIES",
+                              help="Series for charms. (Relevant for recovery test only) [default: xenial]",
+                              default="xenial")
+    pass_through.add_argument("--charm-devel-series", "--devel-series", dest="devel_series", metavar="SERIES",
+                              help="The series to use when testing series in development (Relevant for multi_series_charms only)")
+    pass_through.add_argument("--agent-file",
+                              help="Path to the agent file to be used for bootstrap. (Relevant only for the agent_metadata test)")
+    pass_through.add_argument("--bootstrap-part",
+                              help="Which part of the bootstrap process to test. (Revelant only for the bootstrap test) [default: base]",
+                              choices=['base', 'metadata', 'to'], default="base")
+    pass_through.add_argument("--local-metadata-source",
+                              help="Path to directory with pre-loaded metadata. (Revelant only for the bootstrap test)")
+    pass_through.add_argument("--remote-metadata-source",
+                              help="URL to image metadata (Relevant only for the mixed_images test)")
+    pass_through.add_argument("--export-to",
+                              help="Part to where an exported bundle will be written to. (Relevant only for the bundle_export test)")
+    pass_through.add_argument("--caas-provider",
+                              help="Provider for CaaS models. (Relevant only for caas_deploy_charms test) [default: microk8s]",
+                              choices=["microk8s", "k8s"])
+    pass_through.add_argument('--caas-image-repo',
+                              help="Docker image repo to use. (Relevant only for caas_deploy_charms test) [default: jujuqabot]",
```

Comment:
This is relevant for any caas test. As is --caas-provider etc. We only have one test now but are adding more. Can we reword to be more generic.
Not sure if there's other examples also that are currently a little too specific.

Reply:
Have created a new section.